dluhc-component-library/src/components/maps/MapComponent.tsx

```diff
@@ -7,6 +7,10 @@ import MapContainer from "./components/MapContainer";
 import { Boundary } from "./types";
 
 import "./MapComponent.css";
+import MapLayer from "./components/MapLayer";
```

Comment:
Can we fix the order of imports here please. It should be:

Package imports
Absolute imports
Relative imports (starting with furthest away to closest)

css import